packages/teleterm/src/ui/DocumentGateway/DocumentGateway.tsx

```diff
@@ -71,20 +71,55 @@ export function DocumentGateway(props: State) {
     );
   }
 
+  let cliConnectionString: string;
+
   switch (gateway.protocol as GatewayProtocol) {
     case 'mongodb':
-      return (
-        <Mongo gateway={gateway} title={doc.title} disconnect={disconnect} />
-      );
+      cliConnectionString = useMongo(gateway);
+      break;
     case 'postgres':
-      return (
-        <Postgres gateway={gateway} title={doc.title} disconnect={disconnect} />
-      );
+      cliConnectionString = usePostgres(gateway);
+      break;
     case 'mysql':
-      return (
-        <MySql gateway={gateway} title={doc.title} disconnect={disconnect} />
-      );
-    default:
-      return <Box> {`Unknown protocol type ${gateway.protocol}`}</Box>;
+      cliConnectionString = useMySql(gateway);
+      break;
   }
```

Comment:
As mentioned in the PR description, this will get removed at some point before the preview release in favor of tsh doing this for Teleterm.